example_config.yaml

```diff
@@ -38,3 +40,13 @@ subscriptions:
 data_modeling:
     - space: cc_plant
       lakehouse_abfss_prefix: abfss://FabricCogniteDemo@onelake.dfs.fabric.microsoft.com/CogniteLakehouse.Lakehouse
+
+source:
+    abfss_event_table_path: abfss://ed261bd7-3d7d-4ab5-b70e-a23600c10a62@msit-onelake.dfs.fabric.microsoft.com/ab54a389-a6e8-417f-990f-45edd7d99455/Tables/TestEvents
```

Comment:
Could you change these to use some of the environment variables like LAKEHOUSE_ABFSS_PREFIX or something more generic rather than the specific guids for our lakehouse tables?

Reply:
yeah we could ill change that real quick

Reply:
Actually on second thought let's hold off on that for now. We are going to propose to use the .env file and doing this now would change some stuff on their introduction and may be better to wait to resolve the abfss infection methods